src/System.Diagnostics.Process/tests/ProcessTests.OSX.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Security;
+using Xunit;
+using Xunit.NetCore.Extensions;
```

Comment:
need to check if I can remove unnecessary usings from here.

Reply:
yes, you can always remove unnecessary usings. VS should do it for you in a couple clicks.

Reply:
Surely. But I have VS installed on windows and VSCode on the other machine. 

When I'm using VS on windows, the "remove and sort" feature is not available for me on this file. I think it is because the intellisense for this file is not available as in csproj it is not included for windows.

VSCode also has limited functionality I think. It doesn't have the feature to remove and sort. I think there is limited intellisense functionality available in VSCode. I may be wrong, just my observation so far.

Reply:
Ah right. I wouldn't bother with it then. we don't do it consistently and it's easy for us to do it periodically.

Reply:
@maryamariyan It's definitely not obvious at first, but if you go into the "Configuration Manager" menu in VS, you can switch the build configuration for each project. You can select the OSX configuration for this project and then do refactorings.

Reply:
Not that you need to rush to do that or anything, just a tip for the future 😄

Reply:
Thanks @mellinoe :)